pkg/controller/pxc/backup.go

```diff
@@ -293,6 +293,12 @@ func (r *ReconcilePerconaXtraDBCluster) deletePITR(cr *api.PerconaXtraDBCluster)
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      deployment.GetBinlogCollectorDeploymentName(cr),
 			Namespace: cr.Namespace,
+			Labels: map[string]string{
+				"app.kubernetes.io/name":      "percona-xtradb-cluster",
```

Comment:
Wdyt that we use this opportunity and refactor this a bit so we don't repeat these strings dozens of times across the repo? To have some constants and reuse them, maybe a helper func in some `labels.go` file that will add these common labels.

Reply:
I checked the code. I don't know. Does it really make sense to create separate utils file to store their only one label? 
As I can see the code will become more implicit, more lines of the code and we don't really win something. 
What do you think? I'm ok to add it in terms of coding practice, but I really do not see much motivations for it.

Reply:
What I meant was that instead of having for example `"app.kubernetes.io/name":      "percona-xtradb-cluster",` all over the place, it will be set only in one file and then just invoke a function that will set the labels.

So the idea is that we have these common labels defined in a single place and then just reuse them where they are needed.